pagopa/io-app
# [IOCOM-843] Section Status alert, new messages home

## Short description
This PR adds the Section Status alert on the new messages home.
It also refactors the old `SectionStatus` component (into a new one) to not use the connect version of the React functional component.

|iOS<br/>No url|iOS<br/>Url|
|-|-|
|![Simulator Screenshot - iPhone 15 - 2024-07-08 at 15 47 28](https://github.com/pagopa/io-app/assets/5150343/8e5bbc73-03b5-4cd6-92bd-9e31aa74e581)|![Simulator Screenshot - iPhone 15 - 2024-07-08 at 15 52 12](https://github.com/pagopa/io-app/assets/5150343/3363bba5-bf29-40c4-a9d8-e8ccaa1d36c8)|

|Android<br/>No url|Android<br/>Url|
|-|-|
|<img width="371" alt="Android_standard" src="https://github.com/pagopa/io-app/assets/5150343/35d9104d-5c03-4202-bd7e-9f06585e0100">|<img width="369" alt="Android_moreInfo" src="https://github.com/pagopa/io-app/assets/5150343/ec29d71f-d654-4b5f-9238-0613732fb487">|

## List of changes proposed in this pull request
- New `SectionStatusComponent` which is the pure functional component version of the old one
- `SectionStatusComponent` added to new messages home

## How to test
Using the io-dev-api-server, configure the backend status section to show the warning on the messages home. Make sure to enable both new DS and new messages home and start the application.
